Adds APIM pages docs (#3868)
wso2/docs-apim: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md

# Analytics Pages Introduction

> The new cloud analytics portal consists of several pages where pages are divided based on different functional aspects.

Reviewer: ```suggestion
The new cloud analytics portal consists of several pages, which are divided based on different functional aspects.
```

>  (e.g., traffic, latency). Using these pages you can get the complete business analytics on your API management

Reviewer: ```suggestion
You can use these pages to get the complete business analytics on your API management system. 

See the topics given below to get an overview of the main capabilities of **Analytics Pages**.
```

>   system. This page will serve as the entry point on how to use the cloud analytics portal and educate you on how to

Reviewer: ```suggestion
```

>    get maximum use out of it. 

Reviewer: ```suggestion
```

> We will start off with introducing you to common controls which will govern the pages. Also we will provide an index

Reviewer: ```suggestion
Let's start with the common controls that govern Analytics pages. See the related links in each topic given below for more details.
```

>  to all pages that are present in the solution so that you can refer those pages in detail.   

Reviewer: ```suggestion
```

 
## Common Controls

>  These are the set of common controls that enable you to govern the data shown in pages. Each page will refer these

Reviewer: ```suggestion
These are the common controls that enable you to govern the data shown in Analytics pages. Each page uses these common controls and page-specific controls to visualize the information requested by the user. 

Explained below are these common controls and their usage.
```

>  common controls and page specific controls to visualize the information requested by the user. Following sections

Reviewer: ```suggestion
```

>   will in detail explain each of these common controls and their usage.

Reviewer: ```suggestion
```

  
### Alerts Page

> This page illustrates information related to business alerts issued by the system for your currently selected

Reviewer: ```suggestion
This page shows the information related to business alerts issued by the system for your currently selected
environment and organization. You can use this page as a health monitoring dashboard and make it visible to
the whole team. You can use this page to drill down on each alert and discover possible anomalies or errors in your published APIs.

For more information, see [Alerts Page]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts)
```

>  environment and organization. You can use this page as a health monitoring dashboard and make it visible to whole

Reviewer: ```suggestion
```

>   team. Using this page you can drilldown on each alert and discover possible anomalies or errors in your published

Reviewer: ```suggestion
```

>    APIs. For more information please refer 

Reviewer: ```suggestion
```

>   [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts) documentation.

Reviewer: ```suggestion
```
